engines/director/detection.cpp

```diff
@@ -99,11 +99,14 @@ static const char *directoryGlobs[] = {
 class DirectorMetaEngine : public AdvancedMetaEngine {
 public:
 	DirectorMetaEngine() : AdvancedMetaEngine(Director::gameDescriptions, sizeof(Director::DirectorGameDescription), directorGames) {
-		_singleId = "director";
 		_maxScanDepth = 2;
 		_directoryGlobs = directoryGlobs;
 	}
 
+	virtual const char *getEngineId() const {
```

Comment:
It is virtual here, but not declared virtual in the previous instances. Maybe it makes sense to normalize it?

Reply:
I'd rather do mass change using clang-tidy in another pull request to add the override attributes / remove the virtual attributes rather than hand editing all these files again.